src/python/pants/option/options_bootstrapper.py

```diff
@@ -104,7 +104,10 @@ def create(cls, env=None, args=None):
     short_flags = set()
 
     def filecontent_for(path):
```

Comment:
This function runs over all of the ini files gathered here:
https://github.com/pantsbuild/pants/blob/a009067657de98f16f8ccd4d79068024b6e1cf3d/src/python/pants/option/options_bootstrapper.py#L39-L70

It's ok if it noops on pants.ini - since it's both added to the list implicitly and an optional file, but the rest came from user configuration that essentially asserts these additional ini files exist. If they don't, `filecontent_for` will fail before this change and now the missing config will be silently skipped. Even if this behavior change is deemed good its probably best not to mix into this unrelated change.

Reply:
Good catch! Thanks for looking at this closely. Should be a better scoped fix now.